src/Gateway/PayplugGatewayOney3x.php

```diff
@@ -0,0 +1,311 @@
+<?php
+
+namespace Payplug\PayplugWoocommerce\Gateway;
+
+use Payplug\PayplugWoocommerce\PayplugWoocommerceHelper;
+use libphonenumber\PhoneNumberType;
+use libphonenumber\PhoneNumberUtil;
+
+// Exit if accessed directly
+if (!defined('ABSPATH')) {
+    exit;
+}
+
+/**
+ * PayPlug WooCommerce Gateway.
+ *
+ * @package Payplug\PayplugWoocommerce\Gateway
+ */
+class PayplugGatewayOney3x extends PayplugGateway
+{
+
+    const MIN_ONEY_PRICE = 100;
+    const MAX_ONEY_PRICE = 3000;
+    const ALLOWED_COUNTRY_CODES = ['FR', 'IT', 'GP', 'MQ', 'GF', 'RE', 'YT'];
```

Comment:
To remove. 

En effet, l'Api a été MAJ pour ne plus avoir besoin de simuler des Pays que ce soit en mode **TEST** ou en mode **LIVE**.